arch/sim/Kconfig

```diff
@@ -117,15 +117,35 @@ config SIM_NETDEV
 	select ARCH_HAVE_NETDEV_STATISTICS
 	select SCHED_LPWORK
 	select SIM_WALLTIME
+	---help---
+		Build in support for a simulated network device.
+
+if SIM_NETDEV
+
+choice
+	prompt "Simulated Network Device Type"
+	default SIM_NETDEV_TAP
+
+config SIM_NETDEV_TAP
+	bool "Simulated Network Device with TAP/WPCAP"
 	---help---
 		Build in support for a simulated network device using a TAP device on Linux or
 		WPCAP on Windows.
 
+config SIM_NETDEV_VPNKIT
+	bool "Simulated Network Device with VPNKit"
```

Comment:
Could you update the related section in:
boards\sim\sim\sim\README.txt
boards\sim\sim\sim\NETWORK-LINUX.txt

Reply:
will do

Reply:
done